### DISCUSSION TITLE
Is there a way to have OpenAMP function as a SMP?

### DISCUSSION THREAD

**Discussion by PSIDRACK** [OpenAMP/open-amp#306] [Q&A]
I currently have an application running on a single A53 processor and there are 3 spare A53's available. Can I use OpenAMP to spread the workload among the 4 processors?  any examples i could pull for review? Thanks

**Comment by arnopo** [comment]
Hello @PSIDRACK,
Difficult to answer because it depends on your use cases.If you are running Linux or other OS on the A53...which kind of data do you need to transfer ( control or large buffer of data).   
Could you detail?
